From 4e56d1853997a6229f308ad57a55be336896a80b Mon Sep 17 00:00:00 2001
From: BambooIV <ningdisheng@gmail.com>
Date: Tue, 2 Feb 2016 15:42:46 +0800
Subject: [PATCH 09/50] nx503a: ARM: dts: msm: Add support for ce-device number
 in crypto driver

cherry-pick from 62408f5fe27aba5cb7e7366e167ce300e645b80f

Add an entry for ce-device in qcedev and qcrypto device nodes.

Signed-off-by: BambooIV <ningdisheng@gmail.com>
---
 arch/arm/boot/dts/NX503A/msm8974.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/NX503A/msm8974.dtsi b/arch/arm/boot/dts/NX503A/msm8974.dtsi
index e0bec4d..6856672 100755
--- a/arch/arm/boot/dts/NX503A/msm8974.dtsi
+++ b/arch/arm/boot/dts/NX503A/msm8974.dtsi
@@ -2204,6 +2204,7 @@
 		interrupts = <0 236 0>;
 		qcom,bam-pipe-pair = <1>;
 		qcom,ce-hw-instance = <1>;
+		qcom,ce-device = <0>;
                 qcom,msm-bus,name = "qcedev-noc";
 		qcom,msm-bus,num-cases = <2>;
 		qcom,msm-bus,num-paths = <1>;
@@ -2220,6 +2221,7 @@
 		interrupts = <0 236 0>;
 		qcom,bam-pipe-pair = <2>;
 		qcom,ce-hw-instance = <1>;
+		qcom,ce-device = <0>;
 		qcom,clk-mgmt-sus-res;
                 qcom,msm-bus,name = "qcrypto-noc";
 		qcom,msm-bus,num-cases = <2>;
-- 
1.9.1

